Makefile

```diff
@@ -32,7 +32,7 @@ GOARCH := $(shell go env GOARCH)
 
 GO_LD_FLAGS := -ldflags="-s -w -X github.com/tetratelabs/getenvoy/pkg/version.version=$(GETENVOY_TAG)"
 
-TEST_PKG_LIST ?= ./pkg/...
+TEST_PKG_LIST ?= $(shell go list ./... | grep -v github.com/tetratelabs/getenvoy/test/e2e)
```

Comment:
@llinder this is another trick as most commands about testing have no "exclude" feature. You have to do it externally

Reply:
👍  thanks for the tip!